andela/titan-dev-ah-frontend
# 163518657 Manage notifications

#### What does this PR do?
Adds the ability to control the notifications
#### Description of Task to be completed?
-  Create `Settings` components with React
- Integrate the components with redux for state management
- write unit tests for all elements used by those components 
#### How should this be manually tested?
- clone the repo and check out the following branch `ft-opt-notifications-163518657`
-  Refer to `.sample.env` to update your env file
- run `yarn start` 
-  Navigate `localhost:{port}/settings` to enable or disable notifications 
- run  `yarn test` to  run the unit tests
#### Any background context you want to provide?
You need to be autholized to get access to the data
#### What are the relevant pivotal tracker stories?
#163518657 
#### Screenshots (if appropriate):
## Settings
<img width="1440" alt="Screen Shot 2019-04-17 at 14 24 46" src="https://user-images.githubusercontent.com/25322304/56287445-a400b880-611c-11e9-8fe8-fe042d6603f7.png">
